pkgs/development/python-modules/chispa/default.nix

```
@@ -0,0 +1,23 @@
+{ buildPythonPackage, fetchFromGitHub, lib, poetry, ... }:
+
+buildPythonPackage rec {
+  pname = "chispa";
+  version = "0.8.2";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    repo = "chispa";
+    owner = "MrPowers";
+    rev = "v${version}";
+    sha256 = "sha256-dKiBdJNnPAJQp4OPtUjBPMqZSLNCDhfQ6OwJyHtvTrw=";
+  };
+
+  nativeBuildInputs = [ poetry ];
+
+  meta = with lib; {
+    homepage = "https://github.com/MrPowers/chispa";
+    description = "PySpark test helper methods with beautiful error messages.";
```

There shouldn't be a dot at the end of this:
```suggestion
    description = "PySpark test helper methods with beautiful error messages";
```
Other than that, this lgtm :blush:

---

Thanks for reviewing!